library/general/src/lib/ui/text_helpers.rb

```diff
@@ -53,5 +53,17 @@ def wrap_text(text, wrap = 76, separator: " ", prepend_text: "",
 
       lines.join("\n")
     end
+
+    # Wrap a given text in direction markers
+    #
+    # @param [String] text to be wrapped
+    # @param [String] language code (it gets the current one by default)
+    # @return [String] wrapped text
+    def direct_richtext(text, lang = Yast::Language.language)
+      Yast.import "Language"
```

Comment:
I think this is too late for the parameter declaration.
This method is at an intersection of Language and Text but with more of the language, so maybe we should move it to a place that already import Language?

Reply:
Or we might just move the initialization to the line below.

```ruby
div_with_direction(text, lang = nil)
  Yast.import "Language"
  lang ||= Yast::Language.language
  #...
end
```